packages/block-library/src/template-part/edit/advanced-controls.js

```diff
@@ -2,19 +2,44 @@
  * WordPress dependencies
  */
 import { useEntityProp } from '@wordpress/core-data';
-import { SelectControl, TextControl } from '@wordpress/components';
+import {
+	BaseControl,
+	Button,
+	SelectControl,
+	TextControl,
+	Tooltip,
+} from '@wordpress/components';
 import { sprintf, __ } from '@wordpress/i18n';
 import { InspectorAdvancedControls } from '@wordpress/block-editor';
+import { footer, header, layout } from '@wordpress/icons';
 
 /**
  * Internal dependencies
  */
 import { getTagBasedOnArea } from './get-tag-based-on-area';
 
 const AREA_OPTIONS = [
-	{ label: __( 'Header' ), value: 'header' },
-	{ label: __( 'Footer' ), value: 'footer' },
 	{
+		description: __(
+			'The Header template defines a page area that typically contains a title, logo, and main navigation.'
+		),
+		icon: header,
+		label: __( 'Header' ),
+		value: 'header',
```

Comment:
Not something that needs changed in this PR, but we should centralize these descriptions/definitions so we don't need to re-define them in a handful of places.  I'm trying that as part of https://github.com/WordPress/gutenberg/pull/30395, so a follow up once these both land would be to update this to get that data from the core editor store.